suite-native/module-accounts-import/src/components/AccountImportImportedAccount.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import { useNavigation } from '@react-navigation/core';
+
+import { Box, Button, Divider, Text } from '@suite-native/atoms';
+import { AccountListItem } from '@suite-native/accounts';
+import { Account } from '@suite-common/wallet-types';
+import {
+    AccountsImportStackParamList,
+    AccountsImportStackRoutes,
+    AppTabsRoutes,
+    HomeStackRoutes,
+    RootStackParamList,
+    RootStackRoutes,
+    StackToTabCompositeProps,
+} from '@suite-native/navigation';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { AccountImportSummarySection } from './AccountImportSummarySection';
+
+type AccountImportImportedAccountProps = {
+    account: Account;
+};
+
+type NavigationProp = StackToTabCompositeProps<
+    AccountsImportStackParamList,
+    AccountsImportStackRoutes.AccountImport,
+    RootStackParamList
+>;
+
+const contentWrapperStyle = prepareNativeStyle(() => ({
+    marginBottom: 114,
+}));
+
+export const AccountImportImportedAccount = ({ account }: AccountImportImportedAccountProps) => {
```

Comment:
Can you rename this to `AccountAlreadyImported`? Current name is really confusing.

Reply:
renamed